torch/fx/experimental/fx_acc/acc_ops.py

```diff
@@ -389,30 +388,26 @@ def min_single_tensor_input(*, input):
 def min_two_tensors_input(*, input, other):
     return torch.min(input, other)
 
-@register_acc_op_mapping(
-    op_and_target=("call_function", torch.quantize_per_tensor),
-    arg_replacement_tuples=[
-        ("input", "input"),
-        ("scale", "scale"),
-        ("zero_point", "zero_point"),
-        ("dtype", "dtype"),
-    ],
-    kwargs_to_move_to_acc_out_ty=[
-        ("dtype", "dtype"),
-        ("scale", "q_scale"),
-        ("zero_point", "q_zero_point"),
-    ],
-)
 @register_acc_op
-def quantize_per_tensor(*, input, acc_out_ty=None):
+def quantize_per_tensor(*, input, qparams=None):
```

Comment:
We might still want to use `acc_out_ty` here to be consistent with other quantized ops. Instead we can change how `kwargs_to_move_to_acc_out_ty` works so it would create `qparams` correctly.